.github/buildomat/jobs/test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#:
+#: name = "test"
+#: variety = "basic"
+#: target = "lab-opte-0.22"
+#: rust_toolchain = "stable"
+#: output_rules = [
+#:   "/work/*.log",
+#: ]
+#: skip_clone = true
+#:
+#: [dependencies.xde]
+#: job = "opte-xde"
+#:
+
+#### >>>>>>>>>>>>>>>>>>>>>>>>>>>> Local Usage >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+####
+#### If you are running this locally, you must run the xde.sh job first to have
+#### the artifacts at the expected spot.
+####
+#### <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
```

Comment:
Maybe echo this or something short in the `BUILDOMAT_JOB_ID` cond below?

Reply:
Done.